### ISSUE TITLE
AOT breaks with MD2

### ISSUE THREAD

**Issue by leolorenzoluis** [Promact/md2#118]
```
Error in bail mode: [at-loader] compiled\node_modules\md2\colorpicker\colorpicker.ngfactory.ts:1170:9
    TS2322: Type 'QueryList<any>' is not assignable to type 'Portal<any>'.
  Property '_attachedHost' is missing in type 'QueryList<any>'.
```

When compiling with AOT, there's an error that I get that you can see from above.

**Comment by rosslavery** [comment]
Can confirm the same is happening for me.

Angular 2.4.8.

**Comment by dharmeshpipariya-zz** [comment]
We have fixed this issue in release @0.0.17
